accounts/urls.py

```diff
@@ -1,12 +1,15 @@
 from django.conf.urls import patterns, url, include
 from rest_framework import routers
+from django.contrib import admin
  
-from accounts.user_api import views
- 
-router = routers.DefaultRouter()
-router.register(r'user', views.UserView, 'user')
- 
-urlpatterns = patterns(
-    '',
-    url(r'^', include(router.urls)),
+admin.autodiscover()
+
+urlpatterns = patterns('',
```

Comment:
This is not the correct way to define routes in Django > 1.8. See openstax.urls for the Django 1.9 route declaration format (It should be a list and not use patterns). This format will break in 1.10.